src/main/java/reposense/report/RepoConfigurationJson.java

```diff
@@ -0,0 +1,28 @@
+package reposense.report;
+
+import java.util.Date;
+
+import reposense.model.RepoConfiguration;
+
+/**
+ * Represents the structure of a repository's information in summary.json.
+ */
+public class RepoConfigurationJson {
```

Comment:
quick question: why is this necessary?

Reply:
I believe its because we convert the `String` location variable to a `Location` class, but it is a non-transient variable thus it is printed out in the json, so end up the format in the json output changes, which caused the tests to fail.
In this case, we can just update the expected json files to the new format instead of creating 1 whole new class for this.

Reply:
I was thinking that the json file would be more readable(/editable?) by keeping the fields consistent as it seems unintuitive to have only one field nested.

Reply:
True but let's say if its nested, we can include additional information such as `Local` or `Remote`. 

That would be a benefit to the existing implementation as we can fixed the link to local repository that currently doesn't work for local analysis.